File: 2015/day1/Php/part1/main.php
<?php
parse_str(implode('&', array_slice($argv, 1)), $_GET);
$input = $_GET['input'];
echo "Processing: {$input} \n";
$floor = 0;
$strlen = strlen( $input );
for( $i = 0; $i <= $strlen; $i++ ) {
    $char = substr( $input, $i, 1 );
    if(strcmp($char, "(") === 0) {
        ++$floor;
    } elseif(strcmp($char, ")") === 0) {
        --$floor;
    }
}
echo "{$floor} \n";
?>

File: 2015/day1/Go/part1/main.go
package main

import (
	"fmt"
    "os"
)

func main(){
	input := os.Args[1]
	fmt.Println("Processing ", input)
	floor :=0
	for i := 0; i < len(input); i++  {
		if input[i] == ')' {
			floor++
		} else {
			floor--
		}
	}
	fmt.Println(floor)
}